src/qtpropertybrowser/src/qtpropertybrowser.qrc

```diff
@@ -20,4 +20,4 @@
         <file>images/cursor-whatsthis.png</file>
     </qresource>
 </RCC>
-  
```

Comment:
Please don't change any files in `src/qtpropertybrowser` and `src/qtsingleapplication`. They are from the Qt Solutions repository and changes should be kept minimal in order to make synchronizing the code easier.